src/App.tsx

```diff
@@ -161,10 +166,9 @@ function App() {
         if (!trajectory) {
             return null;
         }
-        const longestAxis = Math.max(viewportSize.width, viewportSize.height);
+        const longestAxis = Math.max(viewportWidth, viewportHeight);
         return new BindingSimulator(trajectory, longestAxis / 3);
-    }, [currentModule, viewportSize, simulationData]);
```

Comment:
Would it be viable to use the fields of the original objects as dependencies here? Though then again... would the linter complain? Again, no need to fight the machine.

Reply:
you're right, and I ended up doing that in another PR, so I'm going to close this PR, thanks for the suggestion!